website/docusaurus.config.js

```diff
@@ -44,6 +44,20 @@ const config = {
     locales: ['en'],
   },
 
+  plugins: [
+    [
+      'content-docs',
+      ({
+        id: 'community',
```

Comment:
I don't want to add a seperate community instance.

Reply:
There will be some documentation of the community to follow. I plan to keep them separate from the project docs.

Reply:
Let's seperate them until we have enough docs.